win32ss/user/ntuser/ntuser.h

```diff
@@ -4,7 +4,7 @@
 #define RETURN(value) { _ret_ = value; goto _cleanup_; }
 #define CLEANUP /*unreachable*/ ASSERT(FALSE); _cleanup_
 #define END_CLEANUP return _ret_;
-
+#define IS_IMM_MODE() (gpsi && (gpsi->dwSRVIFlags & SRVINFO_IMM32))
```

Comment:
Shouldn't this macro need to go into an imm specific header?

Reply:
This macro is specific to `win32k`.